grape-pipeline.nf

```diff
@@ -255,17 +288,17 @@ if ( 'mapping' in pipelineSteps ) {
         }.set { mappingIndex
  }
     } else {
-        genomesForIndex.set { indexGenomes }
-        annotationsForIndex.set { indexAnnotations }
+        Genomes.set { indexGenomes }
+        Annotations.set { indexAnnotations }
     }
-    annotationsForMapping.set { mappingAnnotations }
-    annotationsForInferExp.set { inferExpAnnotations }
-    annotationsForBamStats.set { bamStatsAnnotations }
+    Annotations.set { mappingAnnotations }
```

Comment:
I guess we could get rid of all duplicated channels as channels can be used multiple times in DSL2

Reply:
Done